controllers/humiocluster_defaults.go

```diff
@@ -382,6 +379,10 @@ func (hnp HumioNodePool) GetEnvironmentVariables() []corev1.EnvVar {
 		},
 	}
 
+	for _, defaultEnvVar := range envDefaults {
```

Comment:
It seems to me that this append loop should be present, otherwise why create the envDefaults variable above? Was it missed before or am I misunderstanding something here?